### PR TITLE
spoiler: write the plando settings to the spoiler log

### PR DESCRIPTION
**Pull request by alwaysintreble**
Please format your title with what portion of the project this pull request is
targeting and what it's changing.

ex. "MyGame4: implement new game" or "Docs: add new guide for customizing MyGame3"

## What is this fixing or adding?
adds the enabled plando options to the spoiler

## How was this tested?
generated a few seeds

## If this makes graphical changes, please attach screenshots.
Screenshot_77

### CODE REVIEW

File: BaseClasses.py
Comment (beauxq) [inline BaseClasses.py:1248]: I don't like having a member that is not initialized in `__init__`

I wonder why this info isn't attached to the instance of `MultiWorld`

Comment (alwaysintreble) [reply BaseClasses.py:1248]: i did that at first but didn't like it... the only other alternative i have is passing it as an arg to the init which i also didn't like. i could just assign it in the class body i suppose? probably should just for safety

**Comment by Berserker66** [comment]
We currently never call it "plandomizer"

**Comment by ThePhar** [comment]
@alwaysintreble Can we change the verbiage from `Plandomizer Settings` to `Plando Options`? Since that's what we call it on the website.

**Comment by ThePhar** [comment]
I helped contribute to this PR to move PlandoSettings out of generate, so I'll let another maintainer review.

**Review by Berserker66 — APPROVED** [review]

**Pull request merged**
